falco.yaml

```diff
@@ -749,6 +762,8 @@ webserver:
   # Can be an IPV4 or IPV6 address, defaults to IPV4
   listen_address: 0.0.0.0
   k8s_healthz_endpoint: /healthz
+  # [Incubating] `prometheus_metrics_enabled`
```

Comment:
Am not sure about how to integrate the feature level label here, for a sub-option of a `Stable` config key.